api/openapi-spec/swagger.json

```diff
@@ -2583,9 +2583,14 @@
       }
     },
     "io.argoproj.workflow.v1alpha1.ParallelSteps": {
-      "type": "array",
-      "items": {
-        "$ref": "#/definitions/io.argoproj.workflow.v1alpha1.WorkflowStep"
+      "type": "object",
+      "properties": {
+        "steps": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/io.argoproj.workflow.v1alpha1.WorkflowStep"
+          }
+        }
```

Comment:
This breaks a change in https://github.com/argoproj/argo/pull/2459/.

@mac9416 Could you comment on what you think is wrong here? Don't want to revert your changes unintentionally.

Reply:
The above json matches the _actual_ object structure, but not the _effective_ structure, since ParallelSteps uses a custom marshaller to represent ParallelSteps as an array rather than a struct containing a `steps` array.

This reversion would break argo-client-java, because it would incorrectly marshal API requests and unmarshal API responses.

#2459 added a few lines to _primary.swagger.json to override ParallelSteps' swagger representation. I'll look through this PR to figure out what effectively reverted that change.

Reply:
My `git merge` messed up. I fixed it